tickets/tickets_commands.go

```diff
@@ -27,6 +27,10 @@ const InTicketPerms = discordgo.PermissionReadMessageHistory | discordgo.Permiss
 
 var _ commands.CommandProvider = (*Plugin)(nil)
 
+func TicketsDisabledError(guild *dcmd.GuildContextData) string {
```

Comment:
This probably shouldn't be exported. Also, consider renaming this to `createTicketsDisabledError`, similar to above.